src/Discord.Net.Rest/DiscordRestApiClient.cs

```diff
@@ -1415,6 +1415,9 @@ private static string GetBucketId(BucketIds ids, Expression<Func<string>> endpoi
                 }
 
                 format = builder.ToString();
+                if (format.LastIndexOf('/') == format.Length-1)
```

Comment:
How about checking like this? Might be (insignificantly) faster.

```cs
if (format.Length > 0 && format[format.Length - 1] == '/')
```

Reply:
Better alternative is performing the above check on the *builder* and then doing `builder.Remove(builder.Length - 1, 1)` to avoid the intermediate string allocation.